india_stocks_api/internal/angel/api/data.py

```
@@ -343,111 +364,133 @@ def get_history(self, symbol: str, exchange: str, interval: str,
+                    )
                     current_start = current_end + timedelta(days=1)
                     continue
-                
-                if not response.get('status'):
-                    raise Exception(f"Error from Angel API: {response.get('message', 'Unknown error')}")
-                
+
+                if not response.get("status"):
+                    raise Exception(
+                        f"Error from Angel API: {response.get('message', 'Unknown error')}"
+                    )
```

Ruff (0.14.14)

Abstract `raise` to an inner function

(TRY301)